webapp/security/fixtures/usns.py

```diff
@@ -0,0 +1,29 @@
+#! /usr/bin/env python3
```

Comment:
This file needs `chmod +x`:

``` bash
$ dotrun exec ./webapp/security/fixtures/usns.py /tmp/usns.json

[ $ ./webapp/security/fixtures/usns.py /tmp/usns.json ] ( virtualenv `.venv` )

Traceback (most recent call last):
  File "/snap/dotrun/32/bin/dotrun", line 11, in <module>
    load_entry_point('canonicalwebteam.dotrun==0.0.0', 'console_scripts', 'dotrun')()
  File "/snap/dotrun/32/lib/python3.6/site-packages/canonicalwebteam/dotrun/__init__.py", line 125, in cli
    return dotrun.exec(arguments.remainder or ["bash"])
  File "/snap/dotrun/32/lib/python3.6/site-packages/canonicalwebteam/dotrun/models.py", line 167, in exec
    result = subprocess.check_call(commands, env=env, cwd=self.path)
  File "/snap/dotrun/32/usr/lib/python3.6/subprocess.py", line 306, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/snap/dotrun/32/usr/lib/python3.6/subprocess.py", line 287, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/snap/dotrun/32/usr/lib/python3.6/subprocess.py", line 729, in __init__
    restore_signals, start_new_session)
  File "/snap/dotrun/32/usr/lib/python3.6/subprocess.py", line 1364, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
PermissionError: [Errno 13] Permission denied: './webapp/security/fixtures/usns.py'
```

Reply:
sorry

Reply:
update the qa steps

Reply:
You updated the QA steps to add `python3`, but this seems to me to be the inferior solution. You also added `#! /usr/bin/env python3` to the top of the file, presumably so that it can be executed directly. So why not make it executable?

Reply:
It is executable now, works for me both ways. Just trying to figure out why it does not work.

Reply:
Ah cool. :+1: thanks.